xija/gui_fit/fitter.py

```diff
@@ -139,10 +139,12 @@ def fit(self):
         ui.set_method(self.method)
         ui.get_method().config.update(sherpa_configs.get(self.method, {}))
         ui.load_user_model(CalcModel(self.model), "xijamod")  # sets global xijamod
+        xijamod = globals()["xijamod"]
```

Comment:
Where is this one used?

Reply:
Resolved I think in a later commit though still a little confusing with ui .

Reply:
I really don't understand how these sherpa functions inject into the namespace. It is really unfortunate that `ui.load_user_model` doesn't return the model, but there you go.